subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -216,6 +216,30 @@ static void net_key_list(struct bt_mesh_model *model,
 	k_sem_give(&cli->op_sync);
 }
 
+struct node_reset_param {
+	bool *status;
+};
```

Comment:
No need for this structure, really. The raw `bool *` can be used as an `op_param`.